ecs-java-s3-workshop/build.gradle

```diff
@@ -19,17 +19,15 @@ apply plugin: 'java'
 apply plugin: 'idea'
 apply plugin: 'eclipse'
 
-sourceCompatibility = 1.7
+sourceCompatibility = 1.8
 
 repositories {
     mavenCentral()
 }
 
 dependencies {
-    compile "com.emc.ecs:object-client:3.1.3"
-    compile "com.sun.jersey:jersey-core:1.19"
-}
-
-task wrapper(type: Wrapper) {
-    gradleVersion = '4.3.1'
-}
+    // please change the version of object-client based on your application
+    // v3.x with Jersey 1.x, v4.x or later with Jersey 2.x
+    implementation "com.emc.ecs:object-client:3.4.6"
+    implementation 'commons-codec:commons-codec:1.15'
```

Comment:
better add a commented line with usable Jersey2 dependency version so that it's easier for user to switch for test, such as:
`//implementation "com.emc.ecs:object-client:4.0.0-rc.1"`

BTW, why object-client is insufficient but commons codec needs to be added?